R/test-directory.R

```diff
@@ -181,12 +178,53 @@ test_check <- function(package,
 
 #' @export
 #' @rdname test_dir
+test_local <- function(path = ".",
+                       filter = NULL,
+                       reporter = default_reporter(),
+                       ...,
+                       stop_on_failure = TRUE,
+                       stop_on_warning = FALSE
+                       ) {
+  package <- pkgload::pkg_name(path)
+
+  test_path <- file.path(path, "tests", "testthat")
```

Comment:
Should `test_local()` work from a subdirectory of the package?

Reply:
Good idea, done.